notebooks/community/bigquery_ml/automl_forecasting_bqml_arima_plus_comparison.ipynb

```diff
@@ -0,0 +1,1559 @@
+{
```

Comment:
<p>Update the links</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>